### DISCUSSION TITLE
《开放式组织》如何评价“贡献”

### DISCUSSION THREAD

**Discussion by tisonkun** [OCselected/ttoos#252] [Show and tell]
今天读到《开放式组织》里对“贡献”评价比较疑惑。

> 贡献是决定性因素，但不是交换条件。

英语原文是

> Contribution is critical, but it's not a quid pro quo.

我理解讨论“贡献”作为开源社区当中的社会资本之一的时候，它应该承担一定的货币作用。组织衡量成员的人效，也需要有一个锚定的指标，它可能就是对贡献的某种形式的量化。

但是《开放式组织》里提到贡献“不是交换条件”，我就看不懂了...

**Comment by lijiangsheng1** [comment]
contribution 翻译为贡献不妥，应该为亲力亲为，人们在开源共同体做事，是为了达到自己的目的，而不是以此要换取什么，亲力亲为本身就是收获。

至于是否可以作为社会资本，或者是政治权力，则是另外的机制和决策而定。所以我以为contribution本身不作为条件是正确的。

**Reply by tisonkun** [reply]
明白了。意思是这里的 contribution 强调的是行为而不是接近于 karma 的结果。

**Reply by lijiangsheng1** [reply]
本后是一种工作伦理和社会认知，值得去解释。

**Comment by WillemJiang** [comment]
第一次看原版的时候对engement 感觉不是很深， 后来到自己动手要构建Community的时候发现Community中人与人的网络联系建立的过程就是engement，而建立engement手段就是靠大家看得见摸得着的Contribution，以及Contribution给Community带来的价值。

如果简单把Contribution理解为交换，我觉得会掉进花钱买贡献的怪圈。
